packages/generator/test/renderers/helpers.test.ts

```diff
@@ -127,36 +87,7 @@ describe("Renderer Helpers", () => {
       expect(result[0][1])
         .to.be.an("array")
         .with.lengthOf(1);
-      expect(result[0][1][0]).to.be.an.instanceOf(model.document.Document);
-    });
-  });
-
-  describe("objectFromEntries", () => {
-    type Entries = [string, unknown][];
-
-    it("creates an empty object from an empty list of entries", () => {
-      expect(renderer.objectFromEntries([])).to.deep.equal({});
-    });
-
-    it("creates an object with keys/values matching the list of entries", () => {
-      const entries: Entries = [
-        ["one", 1],
-        ["two", 2]
-      ];
-      const object = { one: 1, two: 2 };
-      expect(renderer.objectFromEntries(entries)).to.deep.equal(object);
-    });
-
-    it("preserves references to object and function values", () => {
-      const nested = { property: "value" };
-      const method = (): null => null;
-      const result = renderer.objectFromEntries([
-        ["nested", nested],
-        ["method", method]
-      ] as Entries);
-      expect(result).to.deep.equal({ nested, method });
-      expect(result.nested).to.equal(nested);
-      expect(result.method).to.equal(method);
+      expect(result[0][1][0]).to.have.keys(["document", "metadata"]);
```

Comment:
The deleted test should remain, and an additional test should be added to test the expected structure of `metadata`.

Reply:
You mean the deleted assertion right? These tests should be deleted as the methods they were testing were removed.